grice/db_controller.py

```diff
@@ -158,6 +158,23 @@ def parse_join(join_str, outer_join: bool):
     return TableJoin(table_name, column_pairs, outer_join)
 
 
+def parse_col_names(column_names):
+    """
+    This method takes a string of comma-seperated column names and returns a list of column names.
+
+    :param column_names: string
+    :return: column_names: list
+    """
+    if column_names:
+        columns_dict = OrderedDict()
```

Comment:
I'm assuming we're using OrderedDict here in order to basically get the functionality of a Set that preserves order, is that correct?